src/ngraph/runtime/gpu/gpu_cuda_kernel_emitters.cpp

```diff
@@ -92,3 +92,39 @@ void runtime::gpu::emit_onehot(std::string name,
                                   0));  // arguments
     CUDA_SAFE_CALL(cuCtxSynchronize()); // Retrieve and print output.
 }
+
+void runtime::gpu::emit_reshape(std::string name,
```

Comment:
const string& for arg. Same comment for other instances of the same thing.

Reply:
Updated.